package.json

```diff
@@ -21,34 +21,35 @@
   "author": "Alex Navasardyan, Ben Holmes",
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.2",
-    "ember-ajax": "0.7.1",
-    "ember-cli": "2.5.0",
+    "broccoli-asset-rev": "^2.4.5",
+    "ember-ajax": "^3.0.0",
+    "ember-cli": "2.13.1",
```

Comment:
Wow thanks ❤️  you're the best.

As long as we're still testing against 1.13.x with the test suite.